### PR TITLE
log on server.stop

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ class Rapptor {
   }
 
   async stop() {
-    await this.server.stop({ timeout: 5 * 1000 });
+    this.server.log(['server', 'stopping', 'sigterm', 'notice'], 'Stopping server...');
```
Comment (jgallen23) [inline index.js]: sigterm didn't necessarily get called here...

File: index.js [hunk 2]
```diff
@@ -53,7 +53,8 @@ class Rapptor {
...
+    await this.server.stop({ timeout: 5000 });
```
Comment (jgallen23) [inline index.js]: doesn't look like we are logging after

File: index.js (hunk 1 above)
Comment (jgallen23) [reply index.js]: ?

File: index.js [hunk 3]
```diff
@@ -53,7 +55,13 @@ class Rapptor {
   }
 
   async stop() {
-    await this.server.stop({ timeout: 5 * 1000 });
+    const tags = ['server', 'stopping', 'notice'];
```
Comment (jgallen23) [inline index.js]: Just append

Comment (orthagonal) [reply index.js]: Done, Github didn't update this review note because the actual change is on the lines below.

**Pull request merged**